db/schema_features.hh

```diff
@@ -22,14 +22,16 @@ enum class schema_feature {
     COMPUTED_COLUMNS,
     CDC_OPTIONS,
     PER_TABLE_PARTITIONERS,
+    SCYLLA_KEYSPACES,
```

Comment:
Why do we need the extra complication of a new "Scylla Keyspaces" table? Couldn't we add more columns to the existing "keyspaces" table? 
Is the thinking that in the future Cassandra might add columns to "keyspaces" too and if they accidentally choose the same name as we did, we'll be screwed? In this case won't it be easier to add columns with "scylla_" in their names?
Or maybe the thinking is that some application may read "keyspaces" and be surprised with the extra columns?

Reply:
Ok, these reasons make sense, so indeed let's go with your separate-system-table approach.

By the way, I noted in my review that even though it is a separate table, it too would need some sort of forward-looking plan, as in the future you may want to add more storage types and options for the existing types, and need to forbid using new types or options until the entire cluster allows them, and so on - so I think some of your code which duplicates the logic of which options exist and which don't in multiple places and maybe should be written in a way that the verification should be done only in one place (e.g., one verification function called by the CQL statement) and other places in the code could just work with string maps without knowledge of which options exist in many different places in the code.